docs/shared/linter-plugin.md

```diff
@@ -0,0 +1,46 @@
+# Linter Plugin
+
+The Linter plugin contains executors, generator, plugin and utilities used for linting JavaScript/TypeScript projects within an Nx workspace.
+
+## Setting Up Linter
+
+### Installation
+
+In any Nx workspace, you can install `@nrwl/linter` by running the following commands if `@nrwl/linter` package is not installed:
+
+```bash
+npm i --save-dev @nrwl/linter
+```
+
+```bash
+yarn add --dev @nrwl/linter
+```
+
+## Lint
+
+You can lint an application or a library with the following command:
+
+```bash
+nx lint my-app
+```
+
+```bash
+nx lint my-lib
+```
+
+## Executors
+
+- [eslint](/linter/eslint) - Lints applications/libraries using [ESLint](https://eslint.org/)
+- [lint](/linter/lint) - [**DEPRECATED**] Lints applications/libraries using [TSLint](https://palantir.github.io/tslint/)
+
+## Generator
+
+- [workspace-rule](/linter/workspace-rule) - Generates a new workspace [ESLint](https://eslint.org/) rule
```

Comment:
You can remove these lines, since the executors and generators are listed on the sidebar